plugins/offers_invreq_hook.c

```diff
@@ -447,7 +447,7 @@ static bool check_payer_sig(struct command *cmd,
 
 	return secp256k1_schnorrsig_verify(secp256k1_ctx,
 					   sig->u8,
-					   sighash.u.u8, sizeof(sighash.u.u8), &payer_key->pubkey) == 1;
+					   sighash.u.u8, sizeof(sighash), &payer_key->pubkey) == 1;
```

Comment:
I think it was more correct before these changes?  Thought this shouldn't change anything on any platform we support...

Reply:
@rustyrussell that change was suggested https://github.com/ElementsProject/libwally-core/pull/321#issuecomment-1090958568 am happy to revert it out if incorrect

What is the best way for me to test?

I'm just playing around on testnet with lnurl, and a lightning addresses implementation so am not running anything mission critical, happy to be a guinea pig